frontend/src/test/scala/bloop/dap/DebugServerSpec.scala

```diff
@@ -267,6 +266,86 @@ object DebugServerSpec extends DebugBspBaseSuite {
     }
   }
 
+  test("supports custom toString for variables") {
```

Comment:
Can we improve the name of this test? Something like "requesting stack traces and variables after breakpoints works".